packages/create-sitecore-jss/src/common/processes/transform.ts

```diff
@@ -192,6 +192,25 @@ export const diffAndWriteFiles = async ({
   }
 };
 
+export const populateEjsData = (answers: BaseArgs, destination?: string) => {
+  const ejsData: Data = {
+    ...answers,
+    helper: {
+      isDev: isDevEnvironment(destination || answers.destination),
+      getPascalCaseName: getPascalCaseName,
+      getAppPrefix: getAppPrefix,
+    },
+  };
+  // calculate relative path between main app and proxy when needed
+  if (answers.proxyAppDestination) {
+    ejsData.helper.relativeProxyAppDestination = path.relative(
```

Comment:
Can we set _proxyAppDestination_? It's not required to mention that it's relative
_proxyAppDestination_ already used in angular-xmcloud/.env

Reply:
I think it's more descriptive this way, separating proxyAppDestination used in bin, init-runner, transformer etc and the path used by helper